docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3'
+    
+services:
+
+  db:
+    image: "postgres"
+    restart: always
+    volumes:
+      - data:/var/lib/postgresql/data
+    expose:
+      - "5432"
+    networks:
+      - internal
+    labels:
+      - "traefik.enable=false"
+    environment:
+      POSTGRES_DB: aletheia
+
+  app:
+    build: .
+    restart: always
+    networks:
+      - proxy
+      - internal
+    expose:
```

Comment:
Should this be `ports`? I don't know if we want to have the host machine be able to access Aletheia or not

Reply:
For testing ports is fine, but Traefik should be able to find the container without it